controllers/events_controller.js

```diff
@@ -1106,3 +1231,36 @@ module.exports.generateCerti = async (req, res) => {
 		sendError(res, "Not eligible for certificate", BAD_REQUEST);
 	}
 };
+
+module.exports.sendEventMails = async (req, res) => {
+	let { type, users, event, subject, content } = req.body;
+	// type: event-reminder, event-followup, event-thanks
+	// users: [{ name, email }]
+	// event: event id
+
+	let events;
+	if (!subject && !content) {
+		events = await Event.findById(event);
+		if (!events) {
+			return sendError(res, "Invalid Event!!", BAD_REQUEST);
+		}
+	}
+
+	let params = {
+			mailType: type
```

Comment:
here we are setting the mailtype to the given type, but what if is is a general email?
In the sendGeneralEmailJob it will console incorrectly with this mailtype, also general emails are having the same mailtype, i.e 'other'

Reply:
condition applied and updated